### PR TITLE
Es/observations with core latest

### PR DESCRIPTION
**Pull request by ErikSin**
User can create, see list, and see map of observations.

# Things to note
- [ ] observations are stored in persisted state using zustand and MMKV
- [ ] useMostAccutrateLocation() hook was created. This hook replaces the logic that was contained in the `LocationContext`

# To do
- [ ] images are not yet saved.

### CODE REVIEW

File: src/frontend/screens/ManualGpsScreen/DdForm.tsx
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/DdForm.tsx:1]: This file has been ported over directly from mapeo.

File: src/frontend/screens/ManualGpsScreen/DmsForm/DmsInputGroup.tsx
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/DmsForm/DmsInputGroup.tsx:1]: this file has been ported over directly from mapeo.

File: src/frontend/screens/ManualGpsScreen/DmsForm/index.tsx
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/DmsForm/index.tsx:1]: this file has been ported over directly from mapeo.

File: src/frontend/screens/ManualGpsScreen/UtmForm.tsx
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/UtmForm.tsx:1]: this file has been ported over directly from mapeo.

File: src/frontend/screens/ManualGpsScreen/index.tsx
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/index.tsx:1]: this file has been ported over directly from mapeo.

File: src/frontend/screens/ManualGpsScreen/shared.ts
Comment (ErikSin) [inline src/frontend/screens/ManualGpsScreen/shared.ts:1]: this file has been ported over directly from mapeo.

File: src/backend/package-lock.json
Comment (achou11) [inline src/backend/package-lock.json:1]: curious why there's a diff for this. there shouldn't be if none of the backend deps have changed

**Review by achou11 — CHANGES_REQUESTED** [review]
A few relatively small questions I wanted to ask before approving - overall seems like changes make sense

**Comment by ErikSin** [comment]
closed as it is being replaced by #157

**Pull request closed without merging**